pipeline/alert.go

```diff
@@ -733,6 +737,65 @@ type HipChatHandler struct {
 	Token string
 }
 
+// Send the alert to JIRA.
```

Comment:
Does anything need to be done on the JIRA side of things to enable this plugin?

If so can you add some steps on how to do so here?